ci/engine.go

```diff
@@ -201,25 +201,23 @@ func (e *Engine) Publish(
 	registryUsername *string,
 	// +optional
 	registryPassword *Secret,
-) (string, error) {
+) ([]string, error) {
 	if len(platform) == 0 {
 		platform = []Platform{Platform(platforms.DefaultString())}
 	}
 
-	ref := fmt.Sprintf("%s:%s", image, e.Dagger.Version)
-	if strings.Contains(e.ImageBase, "wolfi") {
-		ref += "-wolfi"
-	}
-
-	if e.GPUSupport {
-		ref += "-gpu"
```

Comment:
Could we potentially simplify this by moving the logic of what tag names to use into github actions? And then just pass a `tags` parameter that we push to all of.

Reply:
I really like this idea! Let me play with it and see how I can simplify the current approach.